static/app/views/metrics/queryBuilder.tsx

```diff
@@ -109,9 +109,12 @@ export const QueryBuilder = memo(function QueryBuilder({
     [projectIds, projects]
   );
 
-  const tags = useMemo(() => {
-    return uniqBy(tagsData, 'key');
-  }, [tagsData]);
+  const groupByOptions = useMemo(() => {
+    return uniqBy(tagsData, 'key').map(tag => ({
+      key: tag.key,
+      disabled: metricsQuery.query?.includes(tag.key),
```

Comment:
this would also disable the tag if it is in a value, right?
 e.g. the tag `b` if the query is `a:"b"`

Reply:
This is gone now, we still search the query string in order to display an icon [here](https://github.com/getsentry/sentry/pull/68982/files#diff-a631790b36270e25b5c1b2f1aca21e080fb9bb2f7cadf6ab0d0e7b1985e85742R117) but with a `:` and that should work